tests/generate.py

```diff
@@ -0,0 +1,24 @@
+import torch as pt
+
+class Central(pt.nn.Module):
+    def forward(self, pos):
+        return pos.pow(2).sum(dim=1).abs().sum()
```

Comment:
I think this and the similar ones below can be simplified to just `pos.pow(2).sum()`.  Squaring the elements guarantees all of them are positive, so `abs()` does nothing.  And that means the intermediate sum also is unnecessary

Reply:
Good point.

Reply:
Done!